src/OSPSuite.Presentation/Presenters/SimulationOutputMappingPresenter.cs

```diff
@@ -159,9 +143,16 @@ public void InitializeSimulation(ISimulation simulation)
          _simulation = simulation;
       }
 
+      public void UpdateSimulationOutputMappings(SimulationOutputMappingDTO simulationOutputMappingDTO)
```

Comment:
Yeah this is not the right place. As I said, the presenter should not modify the object just because it is dislayign the object. I am going to create an issue for this